application/backend/src/main/resources/apps/typerefinery/components/content/pieChartNonPolar/template.html

```diff
@@ -0,0 +1,11 @@
+<template data-sly-template.pieChartNonpolar="${ @ model }">
+    <div 
+        class="pieChartNonolar"
```

Comment:
Please check the spelling pieChartNonolar

in the CSS it is pieChartNonPolar

Reply:
corrected.